ansible/db_tasks.yml

```diff
@@ -29,15 +29,11 @@
     register: mariadb_repo
     when: ansible_distribution == 'CentOS'
 
-  - name: Install MariaDB Client for CentOS
```

Comment:
Best to leave the old block and add the new installation as "when: ansible_distribution == 'RedHat'" ?